gui/design.lua

```diff
@@ -1475,6 +1475,27 @@ function Design:onInput(keys)
             self.placing_mirror = false
             self.needs_update = true
         else
+            -- Clicking center point
+            if #self.marks > 0 then
+                local center = self.shape:get_center()
+                if pos == center and not self.prev_center then
```

Comment:
`pos == center` is only true if they point to the same table. Is this what you mean, or do you want `same_xyz(pos, center)`?

Reply:
As I understand in this line they are both `Point` objects, and the `==` operator was overloaded with the __eq() function in here:

https://github.com/DFHack/scripts/blob/ead5eaa603a4d7c97908206414fd06155d07a05e/internal/design/util.lua#L77-L81

It is showing the behavior I expect, but I don't understand Lua metatable shenanigans very well, so there could be implications I don't understand.